CHANGELOG.md

```diff
@@ -571,6 +571,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - `isInWeekday` has been renamed to `isWorkday`. [#313](https://github.com/SwifterSwift/SwifterSwift/pull/313) by [kaphacius](https://github.com/kaphacius).
 
 ### Enhancements
+- New **Image** test
```

Comment:
Here is not the place to add new entries, this should be added to the top on next version section ;)

Reply:
whoops.. .-.